packages/reason-relay/src/ReasonRelay.rei

```diff
@@ -48,16 +52,16 @@ type recordSourceRecords;
 
 /**
  * Constants
- */
-
-/**
+ */ /**
```

Comment:
Extremely small nit but maybe this spacing could be preserved? 😅

Reply:
yeah, the quirks of refmt, let me change that :)